modules/ROOT/pages/gov-cloud-features.adoc

```diff
@@ -3,41 +3,42 @@ ifndef::env-site,env-github[]
-|Runtime Manager: xref:runtime-manager::cloudhub.adoc[*CloudHub*]|xref:api-manager::analytics-landing-page.adoc[*Anypoint Analytics*]
-||xref:runtime-fabric::index.adoc[*Runtime Fabric*]
-||xref:mq::index.adoc[*Anypoint MQ*]
-||xref:visualizer::index.adoc[*Anypoint Visualizer*]
-||xref:data-gateway::index.adoc[*Data Gateway*]
-||xref:anypoint-security::index.adoc[*Secrets manager*]
-||xref:anypoint-security::index.adoc[*Anypoint Security*]
-|===
-
-// |xref:design-center::design-create-publish-api-specs.adoc[*Visual API designer*]|Flow designer
-// 2+^h|xref:exchange::index.adoc[*Exchange*]
-// 2+^h|xref:studio::index.adoc[*Anypoint Studio*]
-// |xref:access-management::index.adoc[*Access management*]|Anypoint Monitoring
-// .2+|xref:anypoint-security::index.adoc[*Anypoint Security*]|xref:anypoint-security/tokenization.adoc[Tokenization]
-// |xref:api-manager::latest-overview-concept.adoc[*API Manager*]|Anypoint Analytics
-// |Runtime Manager: xref:runtime-manager::cloudhub.adoc[*CloudHub*]|Runtime Fabric
+Government Cloud includes the full functionality of the Anypoint Platform runtime plane (services and Mule runtime engine) and a subset of the Anypoint Platform control plane (browser-based management of APIs and Mule runtime engines) functionality. Government Cloud supports the full API-led connectivity methodology and end-to-end API lifecycle management.
+
+By default, Mule runtime engine is configured to run in FIPS-compliant mode. See xref:mule-runtime::fips-140-2-compliance-support.adoc[FIPS 140-2 Compliance Support] for additional information.
+
+Although Anypoint Studio is not part of Government Cloud, you can use Anypoint Studio offline to build applications to deploy in Government Cloud.
+
+* xref:design-center::index.adoc[Design Center]
+
+** xref:design-center::design-create-publish-api-specs.adoc[API Designer]
+** xref:design-center::design-create-publish-api-visual-editor.adoc[Visual API Designer]
+** xref:design-center::design-create-publish-api-specs#documenting[API Console]
+** xref:design-center::design-migrating-ms#what-the-mocking-service-is[Mocking service]
+** xref:design-center::design-create-publish-api-specs#designing[RAML and OAS support]
+* xref:exchange::index.adoc[Exchange]
+** xref:exchange::to-describe-an-asset.adoc[API developer portals]
+** xref:exchange::ex2-to-simulate-api-data.adoc[API simulation (mocking service)]
+** xref:exchange::about-documenting-an-api.adoc[API Notebooks]
+** xref:munit::index.adoc[MUnit]
+** xref:apikit::overview-4.adoc[APIKit]
+* Management Center
+** xref:access-management::index.adoc[Access Management]
+** xref:object-store::osv2-guide.adoc[Object Store]
+** xref:api-manager::latest-overview-concept.adoc[API Manager]
+** xref:runtime-manager::index.adoc[Runtime Manager]
+** xref:runtime-manager::cloudhub.adoc[CloudHub]
+
+The following products will be phased into Government Cloud over time:
```

Comment:
Please reword, according to the style reference:

"future functionality or releases
For legal reasons, do not refer to upcoming features or product releases:

Before	This feature will be supported beginning with Mule 5.x.
		This feature is planned for an upcoming release.

After 	This feature is currently not supported."